src/vs/workbench/parts/debug/electron-browser/replEditor.ts

```diff
@@ -30,7 +30,7 @@ export class ReplInputEditor extends CodeEditorWidget {
 		@IContextKeyService contextKeyService: IContextKeyService,
 		@IThemeService themeService: IThemeService
 	) {
-		super(domElement, options, instantiationService, codeEditorService, commandService, contextKeyService, themeService);
+		super(domElement, options, true, instantiationService, codeEditorService, commandService, contextKeyService, themeService);
```

Comment:
Let's postpone turning the flag to `true` here until we get the commands sorted out. Otherwise, I don't think cursor left/cursor right will work in the repl.